dev-client/src/screens/SoilScreen/utils/renderValues.ts

```diff
@@ -30,13 +30,6 @@ export const renderDepthInterval = ({
   return `${label ? `${label}: ` : ''}${start}-${end} cm`;
```

Comment:
issue: This should use i18n.t instead of concatenating strings in code. Also: unit should not be hard-coded.